Frends.CSV.Create/CHANGELOG.md

```diff
@@ -1,10 +1,22 @@
 # Changelog
 
+## [1.2.0] - 2024-05-06
+
+### Improved
```

Comment:
We use https://keepachangelog.com/en/1.0.0/ for changelogs, so impoved should be Added